spec/file_translation/fts_mt.yaml

```diff
@@ -0,0 +1,381 @@
+        finds and returns the file normally, you will receive a `200` SUCCESS
+        response. If you receive any other response status code than `200`, the
+        requested files will not be part of the response.
+
+        When you upload a UTF-16 character encoded file, then /file/get requests
+        for that file will have a character encoding of UTF-16. All other
+        uploaded files will return with a character encoding of UTF-8.
+
+        You can always use the content-type header in the response of a file/get
+        request to determine the character encoding.
+        The output filename is based on the original file name, that was provided during initial upload call.
+        
+        **Note:** for GETTEXT files we're going to return file extension as .po
+        even if during the initial file upload it was uploaded with .pot file extension.
+      tags:
+        - File Machine Translation
+      operationId: downloadMTFile
+      parameters:
+        - name: localeId
+          in: path
+          required: true
+          schema:
+            type: string
+          description: One of translation requested with prior call to File MT end point.
+        - $ref: './fts_common.yaml#/components/parameters/accountUidPathParameter'
+        - $ref: './fts_common.yaml#/components/parameters/fileUidPathParameter'
+        - $ref: './fts_common.yaml#/components/parameters/mtUidPathParameter'
+      responses:
+        200:
+          description: OK
+          content:
+            '*/*':
+              schema:
+                type: string
+                format: binary
+        '400':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error400Response'
+              examples:
+                response:
+                  value:
+                    response:
+                      code: VALIDATION_ERROR
+                      errors:
+                        key: locale.not.requested
+                        message: >-
+                          Processing request did not contain locale de-DE
+                        details:
+                          field: "localeId"
+        '401':
+          $ref: '#/components/responses/Error401ResponseDefinition'
+        '429':
+          $ref: '#/components/responses/Error429ResponseDefinition'
+        '500':
+          $ref: '#/components/responses/Error500ResponseDefinition'
+      x-code-samples:
+        - lang: curl
+          source: >
+            curl -H "Authorization: Bearer $smartlingToken" 
+            https://api.smartling.com/file-translations-api/v2/accounts/$smartlingAccountId/files/$smartlingFtsFileUid/mt/$smartlingMtUid/locales/{$smartlingLocaleId}/file
+
+  mt_get_translated_files:
+    get:
+      summary: Download all MT file (all locales)
+      description: |
+        Download a ZIP archive with all MT for the requested file.
```

Comment:
"Download a ZIP archive of translated files for all requested locales."

"Unlike the "Download translated file (single locale) endpoint..."

Reply:
fixed